notion-distcopy/src/main/scala/com/ambiata/notion/distcopy/Mapping.scala

```diff
@@ -1,18 +1,18 @@
 package com.ambiata.notion.distcopy
 
+import com.ambiata.poacher.hdfs.HdfsPath
 import com.ambiata.saws.s3.S3Address
-import org.apache.hadoop.fs.Path
 
 sealed trait Mapping {
   def render: String
 }
 
-case class DownloadMapping(from: S3Address, to: Path) extends Mapping {
+case class DownloadMapping(from: S3Address, to: HdfsPath) extends Mapping {
   def render: String =
-    s"$from,$to"
+    s"$from,HdfsPath(${to.path})"
```

Comment:
From what i can tell, `render` is only used for logging purposes, but i think it would be nice now that you mention it to change it to `hdfs://...`. I should change the s3 rendering also.